app/models/task.rb

```diff
@@ -8,16 +8,24 @@
 #  job_class              :string(255)      not null
 #  job_first_started_at   :datetime
 #  job_last_failed_at     :datetime
-#  job_retried_at         :datetime
 #  job_succeeded_at       :datetime
 #  job_type_value         :string(255)      not null
 #  num_attempts           :integer          default(0), not null
 #  provider_job_id        :string
+#  taskable_id            :bigint(8)
+#  taskable_type          :string
 #  updated_at             :datetime         not null
 #
+# Indexes
+#
+#  index_tasks_on_taskable_type_and_taskable_id  (taskable_type,taskable_id)
+#
+
 class Task < ApplicationRecord
   TASK_JOB_TYPES = %i(full_fetcher)
 
+  belongs_to :taskable, polymorphic: true
```

Comment:
Is this going to come into use with the "Standard Loan Data Export" story? Asking to check my own understanding of how these will fit together.

Reply:
No, this will likely be used for the first time in the DataExportTask story -- its only job is to allow creating a link between a task and another type of object, this is mostly going to be useful for creating links in the UI to go back and forth between a task and the associated data -- StandardLoanExport is just creating the `process_data` method that fetches the data and creates the 2D Array.